Microsoft.Toolkit.Uwp.Input.GazeInteraction/GazeInput.h

```diff
@@ -215,7 +215,12 @@ public ref class GazeInput sealed
     /// <summary>
     /// Gets the GazePointer object.
     /// </summary>
-    static GazePointer^ GetGazePointer(Page^ page);
+    static _declspec(deprecated("Page parameter is unused and should be omitted")) GazePointer^ GetGazePointer(Page^ page) { return GetGazePointer(); }
+
+    /// <summary>
+    /// Gets the GazePointer object.
+    /// </summary>
+    static GazePointer^ GetGazePointer();
```

Comment:
Is this a breaking change?

Reply:
No, the old signature is still present in the header file with a deprecated tag:

static _declspec(deprecated("Page parameter is unused and should be omitted")) GazePointer^ GetGazePointer(Page^ page) { return GetGazePointer(); }